### DISCUSSION TITLE
Trouble installing DefinitionLibrary package

### DISCUSSION THREAD

**Discussion by MaxBromberg** [nicolaasburgers/definition-library-releases#33] [Q&A] [answered]
I have been unable to install the DefinitionLibrary package successfully. Using commercially lisenced rhino 7, I've installed DefinitionLibrary as in the screenshot, (initially v0.2.3, just tested with v0.2.1) but **no library tab appears** to the right of Help in the topbar, and I also could not find any associated commands through the searching the command bar or settings. 
![PackageManagerDefLibrary](https://github.com/user-attachments/assets/f13a7b5a-ae7d-47df-89eb-9f79ca81c5c6)

Let me know where I went wrong during install, as this package looks great, and just what we need! 

Cheers, 
Max

**Comment by nicolaasburgers** [comment, marked as answer]
Hi Max - just clarifying that DefinitionLibrary is only available in Grasshopper, not the Rhino window, so you won't see any evidence of it being installed in the Rhino menu or commands.
Are you still finding no "Library" menu option appear on the menu bar in Grasshopper, even after restarting Rhino and Grasshopper?

**Reply by MaxBromberg** [reply]
And just when you thought that the clear UI and video tutorials had idiot proofed it... 😅 Thanks!
